src/frontend/package.json

```diff
@@ -8,6 +8,8 @@
     "build-dev": "yarn copy-iframe-resizer; yarn build",
```

Comment:
*in commit text* this PR add/adds

Reply:
*in commit title* I'm not sure how these changes relate for performance as ⚡️would imply

Reply:
it speeds up my development :-) I don;t have to look at the prettier help command to look for the option I need and it should prevent me to have an error on the CI

Reply:
Oh so this is a deliberate misappropriation of the gitmoji spec 👍 I'll allow it!!